code/datums/wires/explosive.dm

```diff
@@ -23,7 +23,7 @@
 
 /datum/wires/explosive/c4/explode()
 	var/obj/item/grenade/plastic/c4/P = holder
-	P.explode()
+	P.prime()

```

Comment:
you're changing functionality here

Reply:
I renamed c4's explode proc to the prime proc which is inherited from grenade so that it could be used by the inherited afterattack. In this case the same proc is being called since the old explode() is now c4's prime(). There are times where the inherited prime was called, running the x4 explosion on c4's rather than the c4 explosion on the c4 however I thought this would end up being for the better either way. Should I be putting smaller details like this in the change log too?

Reply:
thats fair, I won't do that again. I'm going to edit the main post to state this since there was another part of the code that I did pretty much the exact same kind of proc renaming to an inherited version to.